lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -99,18 +99,19 @@ <h3 class="course-title-info" id="course-title">
               <div class="col-2">
                 % if enable_bulk_purchase:
                 <div class="numbers-row" aria-live="polite">
-                  <label for="field_${item.id}">${_('Students:')}</label>
-                  <div class="counter">
-                      <input maxlength="3" class="spin-counter" title="${_('Input quantity and press enter.')}" max="999" type="text" name="students" value="${item.qty}" id="field_${item.id}" data-unit-cost="${item.unit_cost}" data-qty="${item.qty}" data-item-id="${item.id}" aria-describedby="students-${item.id}">
+                  <label for="field_${item.id}">${_('Students:')}&nbsp;<strong>${item.qty}</strong></label>
+                  <!--
+                  <div class="counter" style="padding-right: 10px;">
+                      <input disabled="" maxlength="3" class="spin-counter" title="${_('Input quantity and press enter.')}" max="999" type="text" name="students" value="${item.qty}" id="field_${item.id}" data-unit-cost="${item.unit_cost}" data-qty="${item.qty}" data-item-id="${item.id}" aria-describedby="students-${item.id}">
                   </div>
-                  <button class="inc button" data-operation="inc">
+                   <button class="inc button" data-operation="inc">
                     <span class="icon fa fa-caret-up" aria-hidden="true"><span>+</span></span>
                     <span class="sr">${_('Increase')}</span>
                   </button>
                   <button class="dec button" data-operation="dec">
                     <span class="icon fa fa-caret-down" aria-hidden="true"></span>
                     <span class="sr">${_('Decrease')}</span>
```

Comment:
Do we really need to duplicate changes made to the `theme` repo?

Reply:
What if theme will be broken?

Reply:
Now I've got a lot of work :)